datalad/tests/test_nonasyncrunner.py

```diff
@@ -135,18 +154,21 @@ def test_thread_exit():
 def test_inside_async():
     async def main():
         runner = WitlessRunner()
-        return runner.run(["echo", "abc"], StdOutCapture)
+        return runner.run(
+            (["cmd.exe", "/c"] if on_windows else []) + ["echo", "abc"],
+            StdOutCapture)
 
     loop = asyncio.get_event_loop()
     result = loop.run_until_complete(main())
-    eq_(result["stdout"], "abc\n")
+    eq_(result["stdout"], "abc" + os.linesep)
 
 
 @known_failure_osx
-@known_failure_windows
 @with_tempfile(mkdir=True)
 @with_tempfile
 def test_popen_invocation(src_path, dest_path):
+    if on_windows:
+        raise SkipTest("Test requires not implemented __reduce_ex__ of wrapt's object proxy")
```

Comment:
I would keep this test as is, and just add a comment near decorators -- it is not only windows but OSX as well, according to https://github.com/GrahamDumpleton/wrapt/issues/102#issuecomment-766833709 

```suggestion
```

Reply:
Done.